### PR TITLE
Change project module path from "github.com/vmware-tanzu" to "github.…

### PR DESCRIPTION
**Pull request by shreyassreenivas**
…com/vmware"

Initially the projected was created github.com/vmware-tanzu organisation and later it was moved under github.com/vmware but the module path was not migrated.

1. **What this PR does / why we need it**:

Change project module path

3. **Which issue(s) this PR fixes**

        (optional, in `fixes #<issue number>` format, will close the issue(s) when PR gets merged):

        Fixes #

4. **Additional information**

5. **Special notes for your reviewer**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,6 +8,11 @@ build:
 	mkdir -p ~/.terraform.d/plugins/vmware/dev/tanzu-mission-control/1.0.0/darwin_amd64/
 	cp bin/terraform-provider-tanzu-mission-control_v1.0.0 ~/.terraform.d/plugins/vmware/dev/tanzu-mission-control/1.0.0/darwin_amd64/
 
+build-arm:
```
Comment (shobha2626) [inline Makefile:11]: It's same as make build?

Comment (shreyassreenivas) [reply Makefile:11]: Yes, by default the build will build for amd (which is still the majority). The new build tag is for arm, which is the architecture used by Mac OS

**Review by ishangupta-ds — APPROVED** [review]
LGTM!

File: Makefile (hunk 1 above)
Comment (shobha2626) [reply Makefile:11]: There should be some difference in the path used for arm?

File: Makefile [hunk 2]
```diff
@@ -8,6 +8,11 @@ build:
...
+	go build -o bin/terraform-provider-tanzu-mission-control_v1.0.0
```
Comment (Vasundharashukla) [inline Makefile:12]: Can we add `rm -rf ~/.terraform.d/plugins/vmware/dev/tanzu-mission-control/*` ?

File: Makefile (hunk 1 above)
Comment (shobha2626) [reply Makefile:11]: amd and arm are different

**Review by shobha2626 — APPROVED** [review]
LGTM

**Review by Vasundharashukla — APPROVED** [review]

**Pull request merged**